observations/obs_converters/template/work/quickbuild.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+# DART software - Copyright UCAR. This open source software is provided
+# by UCAR, "as is", without charge, subject to all terms of use at
+# http://www.image.ucar.edu/DAReS/DART/DART_download
+
+main() {
+
+export DART=$(git rev-parse --show-toplevel)
+source "$DART/build_templates/buildconvfunctions.sh"
+
+CONVERTER=$(basename "$(dirname "$PWD")")
```

Comment:
This does not support out of core builds, 
CONVERTER should be set to the CONVERTER name